tests/attr/layer/test_layer_gradient_shap.py

```diff
@@ -112,33 +117,33 @@ def test_basic_multi_input(self):
 
     def _assert_attributions(
         self,
-        model,
-        layer,
-        inputs,
-        baselines,
-        target,
-        expected,
-        expected_delta=None,
-        n_samples=5,
-        attribute_to_layer_input=False,
-        add_args=None,
-    ):
+        model: Module,
+        layer: Module,
+        inputs: TensorOrTupleOfTensors,
+        baselines: Union[TensorOrTupleOfTensors, Callable],
+        target: Union[int, Tuple[int, ...], Tensor, List[Tuple[int, ...]]],
+        expected: Union[Tensor, Tuple[Tensor, ...], List, Tuple[List, ...]],
+        expected_delta: Tensor = None,
+        n_samples: int = 5,
+        attribute_to_layer_input: bool = False,
+        add_args: Any = None,
+    ) -> None:
         lgs = LayerGradientShap(model, layer)
         attrs, delta = lgs.attribute(
             inputs,
-            baselines=baselines,
+            baselines,
             target=target,
             additional_forward_args=add_args,
             n_samples=n_samples,
             stdevs=0.0009,
             return_convergence_delta=True,
             attribute_to_layer_input=attribute_to_layer_input,
         )
-        assertTensorTuplesAlmostEqual(
-            self, attrs, expected, delta=0.005,
-        )
+        assertTensorTuplesAlmostEqual(self, attrs, expected, delta=0.005)
         if expected_delta is None:
             _assert_attribution_delta(self, inputs, attrs, n_samples, delta, True)
         else:
-            for delta_i, expected_delta_i in zip(delta, expected_delta):
-                self.assertAlmostEqual(delta_i, expected_delta_i, delta=0.01)
+            for delta_i, expected_delta_i in zip(
+                torch.unbind(delta), torch.unbind(expected_delta)
```

Comment:
Do we need the unbind calls here? I think this should type hint fine without it if testing with torch 1.4.

Reply:
Cool, yeah, it works w/1.4 without unbinding.